appserver/tests/payara-samples/samples/clustered-singleton/clustered-singleton-test/licenseheader.txt

```diff
@@ -0,0 +1,43 @@
+<#if licenseFirst??>
```

Comment:
This looks like a leaked file from NetBeans, probably needs deleting

Reply:
ok